snap/hooks/configure

```diff
@@ -53,6 +53,28 @@ then
   fi
 fi
 
+if ! is_strict
+then
+  # set sysctl parameters 
+  if ! grep -q '^fs.aio-max-nr=' /etc/sysctl.d/10-microk8s.conf
+  then 
+      echo "fs.aio-max-nr=1048576" >> /etc/sysctl.d/10-microk8s.conf
```

Comment:
What if the user has set this or any of the other limits to something larger somewhere else?

Reply:
How this works is that there are multiple locations with different precedence. The `/etc/sysctl.conf` takes the highest precedence amongst the files. The precedence goes as
* `/usr/lib/sysctl.d/*.conf` and `/etc/sysctl.d/*.conf`. The precedence is set by the prefix number (higher number - higher precedence)
* `/etc/sysctl.conf`

I've set it to lowest priority available, so any other file should override this if there are any customizations in that regard. Although there is a small possibility for anyone to create a file that could have a lower precedence. 

I've thought about checking first with `sysctl -n fs.aio-max-nr` and comparing if the value is larger but this does not guarantee the set number is actually persistent at reboot etc. 

Maybe we can `grep` and check the paths above with value comparisons just to be safe. I'll try to add this.